cli/src/api/mod.rs

```diff
@@ -179,6 +180,12 @@ impl PhylumApi {
             .authenticated)
     }
 
+    /// Get information about the authenticated user
+    pub async fn user_info(&self, auth_info: &AuthInfo) -> Result<UserInfo> {
```

Comment:
The `auth_info` here will always be the same `auth_info` that was passed in to `PhylumApi::new(...)`. I'm not sure there's an easy way to refactor and fix that, but it feels odd to have to pass in the same object twice.